### PR TITLE
fix: improve stabilty of random failing tests

### PR DESCRIPTION
**Pull request by TomMenga**

### CODE REVIEW

File: src/components/sbb-tooltip/sbb-tooltip.tsx [hunk 1]
```diff
@@ -350,7 +350,7 @@ export class SbbTooltip extends LitElement {
         .filter((el) => el instanceof window.HTMLElement)
         .find((el) => (el as HTMLElement).matches(IS_FOCUSABLE_QUERY)) as HTMLElement;
       clearTimeout(this._closeTimeout);
-      await this.close();
+      this.close();
```
Comment (jeripeierSBB) [inline src/components/sbb-tooltip/sbb-tooltip.tsx:353]: If you remove that, function probably doesn't need to be async at all. Please leave a comment why await should not be used here (as e.b. I tend to always fixing not awaited calls ;) )

Comment (TomMenga) [reply src/components/sbb-tooltip/sbb-tooltip.tsx:353]: oh I missed it :/
Nice catch

File: src/components/sbb-tooltip/sbb-tooltip.tsx [hunk 2]
```diff
@@ -343,14 +343,14 @@ export class SbbTooltip extends LitElement {
   };
 
   // Close tooltip on backdrop click.
-  private _closeOnBackdropClick = async (event: PointerEvent): Promise<void> => {
+  private _closeOnBackdropClick = (event: PointerEvent): Promise<void> => {
```
Comment (jeripeierSBB) [inline src/components/sbb-tooltip/sbb-tooltip.tsx:346]:
```suggestion
  private _closeOnBackdropClick = (event: PointerEvent): void => {
```

File: src/components/sbb-tooltip/sbb-tooltip.tsx (hunk 1 above)
Comment (jeripeierSBB) [reply src/components/sbb-tooltip/sbb-tooltip.tsx:353]: Can you add a comment why it should not be awaited?

Comment (TomMenga) [reply src/components/sbb-tooltip/sbb-tooltip.tsx:353]: It was needed because stencil forced the async on the `close()` method.

Comment (jeripeierSBB) [reply src/components/sbb-tooltip/sbb-tooltip.tsx:353]: Ah sorry, the close method is not async anymore

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**